anta/tests/cvx.py

```diff
@@ -87,3 +87,47 @@ def test(self) -> None:
         cluster_status = command_output["clusterStatus"]
         if (cluster_state := cluster_status.get("enabled")) != self.inputs.enabled:
             self.result.is_failure(f"Management CVX status is not valid: {cluster_state}")
+
+
+class VerifyActiveCVXConnections(AntaTest):
+    """Verifies the expected number of active CVX Connections.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the expected number of connections is present.
+    * Failure: The test will fail if the unexpected number of connections is present.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.cvx:
+      - VerifyActiveCVXConnections:
+          expected_connection_count: 100
```

Comment:
Should be connections_count with an `S` ?
```suggestion
          connections_count: 100
```

We generally don't use `expected` in the inputs so you can keep it to `connections_count`